exercises/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+	"unsafe"
+)
+
+func randNum() int {
+	max, min := 10, 1
+	rand.Seed(time.Now().UnixNano())
+	num := rand.Intn((max - min) + min)
```

Comment:
Hmmm... `(max - min) + min` is the same as `max - min + min = max`
did you want smth like `min + rand.Int(max - min + 1)`?

Reply:
I wanted random fibonacci number from sequence from1 to 10 at first, but gave up and now its just a random odd number from 1 to 17